packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -419,7 +419,7 @@ export enum NetworkMode {
  * For tasks that use a bind mount host volume, specify a host and optional sourcePath.
  *
  * For more information, see Using Data Volumes in Tasks:
- * [https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_data_volumes.html]
+ * @see https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_data_volumes.html
```

Comment:
This link will not appear inline. It will be shown at the bottom of the text block, so the colon will be dangling "see Using Data Volumes in Tasks:", and then nothing will follow.

Is that what you want?

If you wanted an inline MarkDown link, you should have written something like: 

```
For more information, see [Using Data Volumes in Tasks](https://docs.a...).   <-- period here
```

Reply:
Addressing, thanks!